earlyoom.service

```diff
@@ -1,7 +1,7 @@
 [Service]
 StandardOutput=null
 StandardError=syslog
-ExecStart=/usr/local/bin/earlyoom
+ExecStart=${PREFIX}/bin/earlyoom
```

Comment:
I guess this could be used as an alternative to keep the service file untouched:
```
sed -i s|/usr/local|$(PREFIX)|g " $(TARGET)/lib/systemd/system/earlyoom.service"
```

It's really hard to find out how other projects solve this problem, either they don't provide a systemd unit at all or use ludicrously big build systems like autoconf and replace the variables at `./configure` step.